packages/jetpack-blocks/webpack.config.js

```diff
@@ -0,0 +1,185 @@
+/** @format */
+/**
+ **** WARNING: No ES6 modules here. Not transpiled! ****
+ */
+/* eslint-disable import/no-nodejs-modules */
+
+/**
+ * External dependencies
+ */
+const _ = require( 'lodash' );
+const path = require( 'path' );
+const fs = require( 'fs' );
+const webpack = require( 'webpack' );
+const CircularDependencyPlugin = require( 'circular-dependency-plugin' );
+const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
+const DuplicatePackageCheckerPlugin = require( 'duplicate-package-checker-webpack-plugin' );
+const Minify = require( '@automattic/calypso-build/webpack/minify' );
+const SassConfig = require( '@automattic/calypso-build/webpack/sass' );
+const TranspileConfig = require( '@automattic/calypso-build/webpack/transpile' );
+const wordpressExternals = require( '@automattic/calypso-build/webpack/wordpress-externals' );
+
+/**
+ * Internal dependencies
+ */
+const cacheIdentifier = require( '../../server/bundler/babel/babel-loader-cache-identifier' ); // FIX ME
+const config = require( '../../server/config' ); // FIX ME
+// const { workerCount } = require( './webpack.common' ); // todo: shard...
+//
+/**
+ * Internal variables
+ */
+const bundleEnv = config( 'env' );
+const isDevelopment = process.env.DEV_CYCLES === 'true';
```

Comment:
`DEV_CYCLES`? This looks off. Maybe:

```suggestion
const isDevelopment = process.env.NODE_ENV !== 'production';
```